minion/plugins/base.py

```diff
@@ -141,7 +142,33 @@ def report_progress(self, percentage, description):
     def report_issues(self, issues):
         if issues:
             for issue in issues:
-                issue['Id'] = str(uuid.uuid4())
+                # Generate a unique Id for each issue
```

Comment:
These should use single quotes to be consistent with the rest of the Minion codebase.